01-Authorization/README.md

```diff
@@ -72,18 +72,32 @@ mvn spring-boot:run
 
 ## Test the API
 
-To test the non-secure endpoint send a `GET` request at `http://localhost:3001/ping`.
+To test the non-secure endpoint send a `GET` request at `http://localhost:3001/login`.
 
 ```bash
-curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" "http://localhost:3001/ping"
+curl -X GET -H "Content-Type: application/json" -H "Cache-Control: no-cache" "http://localhost:3001/login"
 ```
 
-You should get the message: `All good. You DO NOT need to be authenticated to call /ping`.
+You should get the message: `All good. You DO NOT need to be authenticated to call /login`.
 
-To test the secure endpoint send a `GET` request at `http://localhost:3001/secured/ping`. In this case you will also have to add a valid `id_token` to your request.
+To test the secure endpoint send a `GET` request at `http://localhost:3001/secured/ping`. In this case you will also have to add a valid `access_token` to your request.
```

Comment:
@Amialc This endpoint doesn't exist anymore. Any endpoint besides `/login` requires the user to be authenticated, meaning that if you have a valid `access_token` in your header, this call will return a http status code 404 because the endpoint it's not defined in the controller. You can check those [here](https://github.com/auth0-samples/auth0-spring-security-api-sample/blob/master/01-Authorization/src/main/java/com/auth0/example/PhotosController.java). Keep in mind that this is also mentioned below in the line 86.